ocs_ci/ocs/resources/packagemanifest.py

```diff
@@ -70,7 +84,14 @@ def get_default_channel(self):
 
         """
         self.check_name_is_specified()
-        return self.data['status']['defaultChannel']
+        try:
+            return self.data['status']['defaultChannel']
+        except KeyError as ex:
+            log.error(
+                "Can't get default channel for package manifest. "
+                "Value of self.data attribute: " + str(self.data)
```

Comment:
```suggestion
                "Value of self.data attribute: %s", str(self.data)
```

Reply:
Your suggestion is most reasonable way (and the only correct) to use python logging, but in this project, we apparently decided against that, likely on the grounds that we are logging everything on DEBUG level anyway.

Reply:
And I can't even find the past reviews, as they are gone by now. I don't really care anyway.

Reply:
We haven't decided against it according to our [coding guidelines](https://github.com/red-hat-storage/ocs-ci/blob/master/docs/coding_guidelines.md)